DatadogRUM/Sources/RUMMonitorProtocol+Internal.swift

```diff
@@ -7,6 +7,18 @@
 import Foundation
 import DatadogInternal
 
+/// Internal attributes used specifically in RUM, passed from cross-platform bridge.
+/// Used to configure or override SDK internal features and attributes for the need of cross-platform SDKs (e.g. React Native SDK).
+public struct RUMInternalAttributes {
+    /// Custom Flutter vital - First Build Complete. The amount of time between a route change (the start of a view) and when the first
+    /// `build` method is complete. In nanoseconds since view start
+    public static let flutterFirstBuildComplete: String = "_dd.performance.first_build_complete"
+
+    /// Custom value for Interaction To Next view.
+    /// For Flutter this is the amount of time between an action occurring and the First Build Complete ocurring on the next view.
+    public static let customINVValue: String = "_dd.view.custom_inv_value"
+}
+
```

Comment:
convention/ Can we move it to [`CrossPlatformAttributes`](https://github.com/DataDog/dd-sdk-ios/blob/363861da11ca1fe0861010bdcbc1402898f03af5/DatadogInternal/Sources/Attributes/Attributes.swift#L67-L69)?

Reply:
An argument towards continuing existing convention ([CrossPlatformAttributes](https://github.com/DataDog/dd-sdk-ios/blob/363861da11ca1fe0861010bdcbc1402898f03af5/DatadogInternal/Sources/Attributes/Attributes.swift#L67-L69)) is that this will keep these attributes internal by convention (clients do not depend on `DatadogInternal`). Otherwise, if storing it in `DatadogRUM`, they became a public API that needs to be maintained.

With lack of better alternative at the moment (maybe `@_spi(Internal)`?), I'd recommend following the existing way. Even if not perfect, it's maintainable.